src/os/bluestore/BlueStore.cc

```diff
@@ -4953,13 +4953,13 @@ void BlueStore::_init_logger()
 	    "Number of onodes in cache");
   b.add_u64(l_bluestore_pinned_onodes, "bluestore_pinned_onodes",
             "Number of pinned onodes in cache");
-  b.add_u64_counter(l_bluestore_onode_hits, "bluestore_onode_hits",
+  b.add_u64_counter(l_bluestore_onode_hits, "onode_hits",
```

Comment:
@aaSharma14 Are you sure are you using these changed named values in `OSD onode hit ratio grafana panel`?

Reply:
Yes @sunilangadi2 , am sure